### PR TITLE
:sparkles: feat(pagination): add a default pagination component

### PR DESCRIPTION
**Pull request by neolitec**
### Description of the Change

Add default pagination.
This new default pagination requires the presence of `ovh-ui-kit`.

If no custom pagination is defined, this will be the displayed pagination widget.

screen shot 2017-08-16 at 4 43 34 pm

### CODE REVIEW

File: src/pagination/default-pagination.component.js [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+        }
+
+        capitalize (string) {
+            return capitalize(string);
+        }
+    },
+    template: `<div class="pagination">
+    <div class="pagination__text">
+      <div class="pagination__size-selector tui-dropdown">
+        <span role="button"
```
Comment (jimmyfortinx) [inline src/pagination/default-pagination.component.js]: if you want to use it as a `button`, why keeping it as a `span`

Comment (neolitec) [inline src/pagination/default-pagination.component.js]: There is plenty more things to write if I must use a button and reset all default styles of a button than simply using a span as a button...

File: src/pagination/default-pagination.component.js [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+                this.pageSizeToggle($event);
+            } else
+            // Escape
+            if ($event.keyCode === 27) {
+                $event.preventDefault();
+                this.menuPageSizeVisible = false;
+            }
+        }
+
+        pageNumberKeydown ($event) {
```
Comment (Netnix) [inline src/pagination/default-pagination.component.js]: In the future we will need this to work from inside the dropdown.

File: src/pagination/default-pagination.component.js [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+          <i class="oui-icon oui-icon-chevron-down" aria-hidden="true"></i></button>
+          <div class="tui-dropdown-menu"
+            ng-class="{ 'tui-dropdown-menu_visible': $ctrl.menuPageNumberVisible }">
+            <ul class="tui-dropdown-menu__content">
+              <li class="tui-dropdown-menu__item"
+                ng-repeat="page in $ctrl.pagesList">
+                <button class="oui-button"
+                  ng-class="{ 'oui-button_selected': page === $ctrl.currentPage }"
+                  ng-bind="page"
+                  ng-click="$ctrl.setPageNumber(page)"></button>
```
Comment (Netnix) [inline src/pagination/default-pagination.component.js]: I'm not a accessibility pro, but I think you could do something like:
aria-label="$ctrl.capitalize($ctrl.config.words.page) + page + $ctrl.config.words.of + $ctrl.pageCount"
To check.

**Pull request merged**